src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
+@import './variables';
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $secondary-color;
+  border-radius: $border-radius;
+
+    &__image {
+      background-image: url(../images/imac.png);
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-position: center;
+      width: 160px;
+      height: 134px;
+      margin: $spacing-large $spacing-medium;
+    }
+
+    &__title {
+      font-family: $font-family-roboto;
+      font-size: $font-size-medium;
+      font-weight: 500;
+      line-height: 18px;
+      text-align: left;
+      color: $text-color-dark;
+      width: 166px;
+      height: 36px;
+      margin-left: $spacing-medium;
+    }
+
+    &__code {
+      font-family: $font-family-roboto;
+      font-size: $font-size-small;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: left;
+      color: $text-color-light;
+      margin-left: $spacing-medium;
+    }
+
+    &__reviews-wrapper {
+      display: flex;
+      width: 166px;
+      margin-left: $spacing-medium;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    &__reviews {
+      font-family: $font-family-roboto;
+      font-size: $font-size-small;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: right;
+    }
+
+    &__price-info {
+      display: flex;
+      width: 166px;
+      justify-content: space-between;
+      align-items: center;
+      margin-left: $spacing-medium;
+    }
+
+    &__text {
+      font-family: $font-family-roboto;
+      font-size: $font-size-medium;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+    }
+
+    &__price {
+      font-family: $font-family-roboto;
+      font-size: $font-size-large;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+    }
+
+    &__button {
+      width: 166px;
+      height: 40px;
+      margin-bottom: $spacing-medium;
+      margin-left: $spacing-medium;
+      background-color: $primary-color;
+      border-radius: $border-radius;
+
+        &-text {
+          font-family: $font-family-roboto;
+          font-size: 14px;
+          font-weight: 700;
+          line-height: 16px;
+          text-align: center;
+          color: $white;
+
+          &:hover {
+            background-color: $white;
+            border: 1px solid $primary-color;
+          }
+
+          &-text:hover {
+            color: $primary-color;
+          }
```

Comment:
The hover effect for `card__button-text` should be nested within the `card__button` hover effect. This ensures that the hover effect is applied correctly when the button is hovered over, as per the previous review comments.